docs/user/integration/jaeger/README.md

```diff
@@ -0,0 +1,192 @@
+# Integrate with Jaeger
+
+## Overview
+
+The following instructions outline how to use [Jaeger](https://github.com/jaegertracing/helm-charts/tree/main/charts/jaeger) as a tracing backend with Kyma's [TracePipeline](../../03-traces.md).
+
+## Prerequisites
+
+- Kyma version 2.10 or higher as the target deployment environment
```

Comment:
The prerequisites have changed somehow. You need a kubernetes cluster with the telemetry module installed/enabled. Somehow that is even known implicit that you need the telemetry module as we are in the related documentation..

Reply:
Alright, adjusting the 1st line.

If the procedure won't work without the module, then enabling the module is a prerequisite. 
The doc must make sense even if it stands alone, not only in the context of the surrounding documentation.